pl_bolts/models/self_supervised/simclr/simclr_finetuner.py

```diff
@@ -3,54 +3,162 @@
 
 import pytorch_lightning as pl
 
-from pl_bolts.models.self_supervised import SimCLR, SSLFineTuner
-from pl_bolts.models.self_supervised.simclr.transforms import SimCLREvalDataTransform, SimCLRTrainDataTransform
+from pl_bolts.models.self_supervised.simclr.simclr_module import SimCLR
+from pl_bolts.models.self_supervised.simclr.transforms import SimCLRFinetuneTransform
+from pl_bolts.models.self_supervised.ssl_finetuner import SSLFineTuner
+from pl_bolts.transforms.dataset_normalizations import (
+    imagenet_normalization,
+    stl10_normalization,
+    cifar10_normalization
+)
 
 
 def cli_main():  # pragma: no-cover
-    from pl_bolts.datamodules import CIFAR10DataModule, ImagenetDataModule, STL10DataModule
+    from pl_bolts.datamodules import ImagenetDataModule, STL10DataModule, CIFAR10DataModule
 
     pl.seed_everything(1234)
 
     parser = ArgumentParser()
-    parser = pl.Trainer.add_argparse_args(parser)
-    parser.add_argument('--dataset', type=str, help='stl10, cifar10', default='cifar10')
+    parser.add_argument('--dataset', type=str, help='cifar10, stl10, imagenet', default='cifar10')
     parser.add_argument('--ckpt_path', type=str, help='path to ckpt')
-    parser.add_argument('--data_dir', type=str, help='path to ckpt', default=os.getcwd())
-    args = parser.parse_args()
+    parser.add_argument('--data_path', type=str, help='path to ckpt', default=os.getcwd())
+
+    parser.add_argument("--batch_size", default=64, type=int, help="batch size per gpu")
```

Comment:
this shall come from Trainer

Reply:
I really don't like the ```pl.Trainer.add_argparse_args(parser)``` thing. I can't name any args in the code with the same name. If the args are not visible in definition, people can forget it and it feels like too many things going on under the hood.

Reply:
sounds like you don't like PL :D

> I can't name any args in the code with the same name.

why you could not?

> If the args are not visible in definition, people can forget it

you can simply ask what arguments are available with `--help` what is the problem with it?